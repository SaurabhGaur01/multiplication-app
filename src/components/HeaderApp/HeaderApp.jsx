import React from "react";

const HeaderApp = () => (
    <header className="appHeader">
        <h3 className="title text-white pl-3 pt- pb-2">Multiplication App</h3>
    </header>
)

export default HeaderApp;