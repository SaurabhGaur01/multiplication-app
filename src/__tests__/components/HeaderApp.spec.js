import React from 'react';
import { shallow } from 'enzyme';
import HeaderApp from '../../components/HeaderApp/HeaderApp';

describe('HeaderApp', () => {
    it('should render correctly', () => {
        const renderedModule = shallow(<HeaderApp />);
        expect(renderedModule).toMatchSnapshot();
    });
})