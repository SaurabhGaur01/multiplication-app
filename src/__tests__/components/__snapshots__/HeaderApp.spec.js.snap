// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderApp should render correctly 1`] = `
<header
  className="appHeader"
>
  <h3
    className="title text-white pl-3 pt- pb-2"
  >
    Multiplication App
  </h3>
</header>
`;
